joomla/joomla-cms
# J Installation: loss of session when getting to the "remove" view

Pull Request for Issue #25130 .

### Summary of Changes
Fixes the session handler to be consistent during the installation (in both name and handler) so that before and after the configuration file is written you preserve your session

### Testing Instructions
Copy down the french language pack for the installation directory from Joomla 3. Run the installer in french and check after installation on the success page that Joomla is still in french (obviously there will be some strings in english/missing things as you've copied the old 3.x pack)

### Documentation Changes Required
None
